lib/nat/structs.go

```diff
@@ -26,55 +26,43 @@ type Mapping struct {
 	mut          sync.RWMutex
 }
 
+// Must be called with m.mut locked.
 func (m *Mapping) setAddress(id string, address Address) {
```

Comment:
The convention in Syncthing's code base is to suffix functions that must be called under a lock with `Locked`, makes it very obvious and hard to forget if calling from some place. Same applies to the verify and acquire functions above.

Reply:
Done, comments removed.